setup.py

```diff
@@ -51,6 +51,7 @@ def find_version(*file_paths):
         "colorama>=0.4.1",
         "docker>=4.3.1",
         "ordered-set>=4.0.2",
+        "cfn-lint>=0.35.0",
```

Comment:
any particular reason for this version btw? Thinking we want the latest version since older versions bundle outdated static data (assuming [`0.35.0`](https://github.com/aws-cloudformation/cfn-python-lint/releases/tag/v0.35.0) was just the latest version when this work started since nothing too notable about that release)
```suggestion
        "cfn-lint>=0.43.0",
```